docs/V1_0_UPGRADE_GUIDE.md

```diff
@@ -0,0 +1,9 @@
+# Version 0.17.0 Upgrade Guide
```

Comment:
This file isn't really necessary for simple flag changes like this. v0.16 had major changes to metric names, which is why we created a separate file.

Also, the filename is `V1_0_...` which is incorrect.

Let's just document this under breaking changes.

Reply:
I've removed the file.